### PR TITLE
Add Geist Mono Font

### PR DESCRIPTION
**Pull request by codejagaban**
This PR adds [Geist Mono ](https://github.com/vercel/geist-font) to the list of font files

### CODE REVIEW

File: geist-mono/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "weights": [100, 200, 300, 400, 500, 600, 700, 800, 900],
+  "styles": ["normal"],
+  "defSubset": "latin",
+  "variable": false,
+  "lastModified": "2023-12-19",
+  "version": "v1.1.0",
+  "category": "monospace",
+  "license": {
+    "type": "OFL-1.1",
+    "url": "https://openfontlicense.org/",
```
Comment (ayuhito) [inline geist-mono/metadata.json:14]: Should update this to [Github link](https://github.com/vercel/geist-font/blob/main/LICENSE.TXT).

**Review by ayuhito — CHANGES_REQUESTED** [review]
Looks good, thanks for the PR! Just some changes needed for the license attributions. Can we also move the folder from the root of the monorepo to `fonts/other`?

**Comment by 0xJamin** [comment]
> Looks good, thanks for the PR! Just some changes needed for the license attributions. Can we also move the folder from the root of the monorepo to `fonts/other`?

Hi @ayuhito  Changes updated

**Comment by ayuhito** [comment]
Sorry for the late review. The tests are not passing since the folder is missing a LICENSE file. Can you copy over the license file from the original repository? That should clear the tests.

**Comment by 0xJamin** [comment]
Hi @ayuhito  I have updated the LICENSE

**Review by ayuhito — APPROVED** [review]
Sorry for the delay. I ended up travelling. Thanks for the PR!

**Pull request merged**